webpack/assets/javascripts/react_app/components/common/forms/NumericInput.js

```diff
@@ -15,17 +15,17 @@ const TextInput = ({
   minValue,
 }) => (
   <CommonForm label={label} className={`common-numericInput ${className}`}>
-    <NumericInput
+    <ReactNumericInput
       format={format}
       min={minValue}
       value={value}
       precision={precision}
-      onChange={onChange}
+      onChange={v => onChange({ target: { value: v } })}
```

Comment:
Not sure this is a way to go, but I would like to have the same behavior as other inputs.

Reply:
It feels like an hack, ticking the `onChange` it's an actual element while it's only contain the value.
Does the `onChange` implementation really need the `event` itself?
In other words, can we chnage to "other" calls to the `onChange` so they will do `onChange={e => onChange(e.target.value)}`

Reply:
I were thinking about that and it would be a way to go for me.
Only problem for me is that default <input> onChange is firing the event.
But we are wraping all the inputs anyway, so should be possible to do what you are saying.
Just maybe what about make other function? Like `onValueChange`

Reply:
`onValueChange` sound perfect to me +1, the consumer will know they are only receiving value changes.

Reply:
I agree, in my opinion this should remane ` onChange={onChange}` 
and the parent component that passes this prop should be refactored.

Reply:
Thanks guys :+1: will do just that :)